packages/zilliqa-js-blockchain/src/util.ts

```diff
@@ -35,6 +40,15 @@ export function toTxParams(
     ...rest
   } = <TransactionObj>response.result;
 
+  const msg: TransactionErrorMessageObj = {};
+  if (receipt.errors) {
+    const messageList = Object.values(receipt.errors)
+      .flat(2)
+      .map((cur) => TransactionError[cur]);
+    const errDepth = Object.keys(receipt.errors)[0];
```

Comment:
Let's avoid using `[0]`.
I think we can use` reduce` like the following:
```
const msg = receipt.errors ? Object.keys(receipt.errors).reduce((acc, depth) => {
    const errorMsgList = receipt.errors[depth].map((num) => TransactionError[num])
    return { ...acc, [depth]: errorMsgList }
}, {}) : {}
```

Reply:
Many thanks @neutiyoo, just updated it.